node/src/components/rpc_server/rpcs/account.rs

```diff
@@ -95,3 +103,75 @@ impl RpcWithParams for PutDeploy {
         }
     }
 }
+
+/// Params for "account_put_transaction" RPC request.
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct PutTransactionParams {
+    /// The `Transaction`.
+    pub transaction: Transaction,
+}
+
+impl DocExample for PutTransactionParams {
+    fn doc_example() -> &'static Self {
+        &PUT_TRANSACTION_PARAMS
+    }
+}
+
+/// Result for "account_put_transaction" RPC response.
+#[derive(PartialEq, Eq, Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct PutTransactionResult {
+    /// The RPC API version.
+    #[schemars(with = "String")]
+    pub api_version: ProtocolVersion,
+    /// The transaction hash.
+    pub transaction_hash: TransactionHash,
+}
+
+impl DocExample for PutTransactionResult {
+    fn doc_example() -> &'static Self {
+        &PUT_TRANSACTION_RESULT
+    }
+}
+
+/// "account_put_transaction" RPC
+pub struct PutTransaction {}
+
+#[async_trait]
+impl RpcWithParams for PutTransaction {
+    const METHOD: &'static str = "account_put_transaction";
```

Comment:
Can you request the ticket to cover these new endpoints with `casper-test`?

Reply:
Good point.  Done now.